src/main/java/uk/gov/ch/oauth/tokens/SignInInfoMap.java

```diff
@@ -3,10 +3,49 @@
 import java.util.HashMap;
 import java.util.Map;
 import uk.gov.companieshouse.session.SessionKeys;
+import uk.gov.companieshouse.session.model.AccessToken;
 import uk.gov.companieshouse.session.model.SignInInfo;
+import uk.gov.companieshouse.session.model.UserProfile;
 
+/**
+ * {@inheritDoc} Implementation of SignInInfo used to create map structure matching session map.
+ * Requires OAuthToken and UserProfileResponse implementations of AccessToken and UserProfile in
+ * order to access additional methods.
+ */
 public class SignInInfoMap extends SignInInfo {
 
+    /**
```

Comment:
Have you changed this again or is this coming from the oauth library? I feel like I have already reviewed this before

Reply:
Correct, as they were raised against the OAuth PR after the docs PR was merged, I never got a chance to make them here as well.
Changes where backported to make sure this works with the latest version of the code.